numpy/core/src/common/simd/avx512/operators.h

```diff
@@ -256,4 +256,16 @@
 #define npyv_cmpge_f32(A, B)  _mm512_cmp_ps_mask(A, B, _CMP_GE_OQ)
 #define npyv_cmpge_f64(A, B)  _mm512_cmp_pd_mask(A, B, _CMP_GE_OQ)
 
+// Create mask from the most significant bit of each 8-bit element
+// AVX512F & AVX512BW
+NPY_FINLINE npy_uint64 npyv_movemask_b8(npyv_b8 mask)
+{
+#ifdef NPY_HAVE_AVX512BW
+    return (npy_uint64)_cvtmask64_u64(mask);
+#else
```

Comment:
```suggestion
#ifdef NPY_HAVE_AVX512BW_MASK
    return (npy_uint64)_cvtmask64_u64(mask);
#elif defined(NPY_HAVE_AVX512BW)
    return (npy_uint64)mask;
#else
```
#17129 define NPY_HAVE_AVX512BW_MASK if extra BW mask operations are avalivble.
It seems safe to me that you can cast to uint64 directly without the need to `_cvtmask64_u64`, almost all compilers support that even msvc 2017.